### DISCUSSION TITLE
Can I set up a system based on multi-user streaming?

### DISCUSSION THREAD

**Discussion by bbhxwl** [owncast/owncast#1969] [General Owncast discussion]
Can I set up a system based on multi-user streaming?

You want to stream multiple live broadcasts and pull multiple videos at the same time.

**Comment by gabek** [comment]
Owncast is single-user software. You can run as many instances of Owncast as you like, however, if you wanted to operate multiple streams.

**Reply by bbhxwl** [reply]
> 

Will multi-user systems be considered in the future? And support Chinese?

**Reply by gabek** [reply]
Owncast will never be multi-user software. See the previous discussions about localization and translations to see the state of that.

**Reply by bbhxwl** [reply]
> Owncast will never be multi-user software. See the previous discussions about localization and translations to see the state of that.

I use the live broadcast service of Tencent cloud. The delay is about 1 second. Is this difficult to achieve?
